dev/io.openliberty.io.opentelemetry.2.1/src/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
```

Comment:
This class is identical to the one in 2.0, we could think about finding a way to merge them or use inheritance but its not something I'll block delivery over.